src/store/reducers/settings.js

```diff
@@ -30,7 +30,7 @@ export const initialState = {
         ...defaultUserSettings,
         ...userSettings,
         [THEME_KEY]: readSavedSettingsValue(THEME_KEY, 'light'),
-        [INVERTED_DISKS_KEY]: readSavedSettingsValue(INVERTED_DISKS_KEY) === 'true',
+        [INVERTED_DISKS_KEY]: readSavedSettingsValue(INVERTED_DISKS_KEY, 'false'),
```

Comment:
We have `defaultUserSettings` in constants with default theme and inverted disks values. Probably it will be better to use them as default values in `readSavedSettingsValue`

Reply:
I noticed that you deleted default settings in other PR. So, it seems resolved)